stylelint-polaris/index.js

```diff
@@ -71,7 +73,7 @@ const stylelintPolarisCoverageOptions = {
       /--p-duration-1-5-0/,
     ],
   },
-  typography: {
+  font: {
```

Comment:
`font` aligns with our token group but I'm not sure it describes the full scope of what we're able to lint regarding typography. That said, I'm not attached to `typography` as a category so feel free to proceed with this update!

Reply:
I agree! I'm gonna leave it as typography, as I also wonder how me might want to iterate on or add to this category to account for the new `Text` component folks should be leaning on primarily going forward 🤔